rpc/ws.go

```diff
@@ -90,42 +90,7 @@ func (s *wsServer) handleRequest(conn *websocket.Conn, req *rpctypes.Request) er
 		}
 
 		return s.handleSigner(s.ctx, conn, params.OfferID, params.EthAddress, params.XMRAddress)
-	case rpctypes.SubscribeNewPeer:
-		return errUnimplemented
```

Comment:
Not sure what this was supposed to do, but I assume we can just add it back later if we decide that we need it.